pkgs/by-name/in/inotify-info/package.nix

```
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "inotify-info";
+  version = "unstable-2024-01-06";
+
+  src = fetchFromGitHub {
+    owner = "mikesart";
+    repo = pname;
+    rev = "a7ff6fa62ed96ec5d2195ef00756cd8ffbf23ae1";
+    sha256 = "sha256-yY+hjdb5J6dpFkIMMUWvZlwoGT/jqOuQIcFp3Dv+qB8=";
+  };
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    cp "_release/inotify-info" "$out/bin/"
```

`runHook` should be used in phases like this one. If a user wants to override this package to add a `preInstall` or `postInstall` phase, then that phase will be called if it exists.
```suggestion
    runHook preInstall
    install -Dm755 _release/inotify-info $out/bin/inotify-info
    runHook postInstall
```